frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -268,14 +343,20 @@ export function createYAxisOptions({
 
             const displayDepth = Math.min(depth, MAX_DEPTH);
 
-            const paddedName = " ".repeat(displayDepth * 8) + name;
+            const paddedName = formatCellLabel(
+              name,
+              Y_AXIS_CHART_WIDTH_PX - 75, // scale based on y-axis width
+              "bold 12px sans-serif", // prevents selected style from overlapping count
```

Comment:
nit: Could we avoid hardcoding this by using `SELECTED_STYLE` defined in line 152? It doesn't specify the font size, but maybe it should. I leave this up to you.